docs/dictionary/command/unlock-messages.lcdoc

```diff
@@ -34,5 +34,5 @@ unlock recent (command), property (glossary), handler (glossary),
 message (glossary), command (glossary), execute (glossary), card (object),
 lockMessages (property)
 
-Tags: objects
+Tags: objects, messages
```

Comment:
I think the `objects` tag can probably be removed here - what do you think? I'm not sure why it would have been applied to lock messages in the first place.